src/routes/fetch-posts.e2e.test.js

```diff
@@ -0,0 +1,178 @@
+const { expect } = require('chai')
+const { fixtures, resetDBTable } = require('../../test/utils')
+
+
+describe('GET /feed', () => {   
+
+	let user
+	let accessToken
+
+	before(async ()=> {
+
+		user = await fixtures.insertUser({
+			role : 'admin'
+		})
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+	})
+
+	describe('Failure', ()=> {
```

Comment:
I tried with "true" or "false" but surprisingly I got 200 status code even though both are strings. What do you think might be the cause of that?

Reply:
yea so everything that comes from the query is basically a string, eg. `http://localhost:3000/carl?isDeveloper=true&language=english` , its a string from the start, because there's no way to distinguish between a string, boolean, number from that because they are all provided the same way

so its likely joi that converts it because the schema has been defined as the parameter as `boolean`. 

Try using any other values and let's see

Reply:
yeah and when i used other values I got 404 Bad request